README.md

```diff
@@ -1,102 +1,97 @@
-augur-core
-----------
+# augur-core
 
 [![Build Status](https://travis-ci.org/AugurProject/augur-core.svg)](https://travis-ci.org/AugurProject/augur-core)
 
-### Ethereum contracts for a decentralized prediction market platform
+Ethereum contracts for a decentralized prediction market platform.
 
-Depends on Serpent and Pyethereum
+## Installation
 
-# Installation
+You should already have a system-wide installation of Python and it should be
+Python 2.7.
 
-You should already have a system-wide installation of python, and it should be set to python-2.7.x.
+First install the dependencies, which include PyEthereum (the tool used to test
+Ethereum smart contracts from Python scripts) and the Serpent smart contract
+programming language:
 
-First set up pyethereum, which includes the tools used to test Ethereum smart contracts from python scripts, and the serpent smart contract programming language. <br>
-(can be done as your preferred user from your home directory)
-
-Pyethereum & Serpent Installation:
 ```
 sudo pip install -r requirements.txt
 ```
 
+(If installed outside of a Python virtualenv Serpent will be installed to
+`./src/serpent`. This isn't ideal but won't hurt anything.)
```

Comment:
Do you or @tinybike or @joeykrug have experience releasing packages to PIP?  Is this something we could do with write-access to the GitHub repo?  Maybe we can just ask Vitalik for the keys to publishing the PIP package (however that works)?  I suspect that if we figure out what needs to be done to release a new version, then we could just ask the appropriate parties for permission to execute on that and get an updated version onto PIP.

Barring that, I like option 3 (install into `.deps`).